german/Leichen/Leichen.md

```diff
@@ -23,3 +23,5 @@ Meine Kleidung hindert mich absolut am Schwimmen.
 Die Kälte des Wassers macht meine Bewegungen steif,
 die vollgesogene [Kleidung](Kleidung/Kleidung.md) macht sie träge.
 Irgendwie muss ich versuchen sie los zu werden!
+
+Die Situation scheint meinen Verstand vernebelt zu haben. Die Kälte und das Wasser scheint ihr übriges zu tun. Aber in einem lichten Moment erinnere ich mich an meine [Superkraefte](Superkraefte/Superkraefte.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.